README.md

```diff
@@ -34,10 +34,10 @@ You can setup this fully functional reference architecture (which includes test
 [![Launch Cloud Shell](https://azurecomcdn.azureedge.net/mediahandler/acomblog/media/Default/blog/launchcloudshell.png "Launch Cloud Shell")](https://shell.azure.com/bash)
 1) Download the OEA framework setup script and framework assets to your Azure clouddrive \
 `cd clouddrive`\
-`wget https://github.com/microsoft/OpenEduAnalytics/releases/download/v0.8rc1/OEA_v0.8rc1.zip`\
-`unzip ./OEA_v0.8rc1.zip`
+`wget https://github.com/microsoft/OpenEduAnalytics/releases/download/v0.8rc1/OEA_v0.8.zip`\
```

Comment:
Shouldn't this be 0.8 as well? i.e https://github.com/microsoft/OpenEduAnalytics/releases/download/**v0.8**/OEA_v0.8.zip